tests/testthat/test-mosaic.R

```diff
@@ -357,6 +357,36 @@ test_that("mosaic.dt() returns an appropriately sized data.table", {
 })
 
 test_that("mosaic() returns appropriately formatted json", {
+  map <- data.frame('id' = c('entity.binB', 'entity.binA', 'entity.cat4'),
+                    'plotRef' = c('yAxisVariable', 'xAxisVariable', 'facetVariable1'),
+                    'dataType' = c('STRING', 'STRING', 'STRING'),
+                    'dataShape' = c('CATEGORICAL', 'CATEGORICAL', 'CATEGORICAL'), stringsAsFactors=FALSE)
+
+  df <- testDF[testDF$entity.cat4 == 'cat4_a',]
```

Comment:
testDF also has an `entity.cat1` column which only has 1 value

Reply:
very nice. didnt realize that. ill update.